### ISSUE TITLE
Error in py_call_impl(callable, dots$args, dots$keywords)

### ISSUE THREAD

**Issue by shinhongwu** [magnusfurugard/featuretoolsR#14]
Hi there,

I've tried this package as instructed in README.md but got the error message after executing
```
ft_matrix <- es %>%
  dfs(
    target_entity = "set_1", 
    trans_primitives = c("and", 'divide')
  )
```

error message
```
' Error in py_call_impl(callable, dots$args, dots$keywords) : 
  ValueError: ('Unknown transform primitive divide. ', 'Call ft.primitives.list_primitives() to get', ' a list of available primitives') '
```

What went wrong?

Here's my sessioninfo:
```
R version 3.5.1 (2018-07-02)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=Chinese (Traditional)_Taiwan.950  LC_CTYPE=Chinese (Traditional)_Taiwan.950   
[3] LC_MONETARY=Chinese (Traditional)_Taiwan.950 LC_NUMERIC=C                                
[5] LC_TIME=Chinese (Traditional)_Taiwan.950    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] bindrcpp_0.2.2       magrittr_1.5         featuretoolsR_0.1.0  RevoUtils_11.0.1     RevoUtilsMath_11.0.0

loaded via a namespace (and not attached):
 [1] reticulate_1.12    tidyselect_0.2.4   reshape2_1.4.3     purrr_0.2.5        splines_3.5.1     
 [6] lattice_0.20-38    colorspace_1.3-2   generics_0.0.2     stats4_3.5.1       yaml_2.2.0        
[11] survival_2.44-1.1  prodlim_2018.04.18 rlang_0.3.4        ModelMetrics_1.1.0 pillar_1.4.1      
[16] glue_1.3.0         withr_2.1.2        foreach_1.5.0      bindr_0.1.1        plyr_1.8.4        
[21] lava_1.6.5         stringr_1.4.0      timeDate_3043.102  munsell_0.5.0      gtable_0.3.0      
[26] recipes_0.1.5      devtools_1.13.6    codetools_0.2-16   memoise_1.1.0      caret_6.0-80      
[31] class_7.3-14       Rcpp_0.12.18       scales_0.5.0       ipred_0.9-6        jsonlite_1.5      
[36] ggplot2_3.0.0      digest_0.6.19      stringi_1.1.7      dplyr_0.7.6        grid_3.5.1        
[41] tools_3.5.1        lazyeval_0.2.1     tibble_1.4.2       crayon_1.3.4       pkgconfig_2.0.2   
[46] MASS_7.3-50        Matrix_1.2-17      lubridate_1.7.4    gower_0.1.2        assertthat_0.2.1  
[51] rstudioapi_0.10    iterators_1.0.10   R6_2.4.0           rpart_4.1-13       nnet_7.3-12       
[56] nlme_3.1-137       compiler_3.5.1    
```

**Comment by magnusfurugard** [comment]
Thanks for filing the issue. This seems to be the result of an updated `FeatureTools` version where the `divide` primitive no longer exists. 

Can you verify it runs fine with the following?
```
ft_matrix <- es %>%
  dfs(
    target_entity = "set_1", 
    trans_primitives = c("and")
  )
```

(You can add any primitives from `list_primitives()` in the `trans_primitives`-parameter)

**Comment by magnusfurugard** [comment]
Nevermind, looks like this is an issue with `r_to_py` requiring a list. 

In `dfs` the following lines needs an update
```
[...]
    agg_primitives = reticulate::r_to_py(agg_primitives),
    trans_primitives = reticulate::r_to_py(trans_primitives),
[...]
```

`agg_primitives` and `trans_primitives` needs to be wrapped in `list(...)` if not null.

**Comment by magnusfurugard** [comment]
Fix in a487bb78f8dea8df24a54dfa7e25de8f172aae04 if you want to try it out.

**Comment by magnusfurugard** [comment]
Resolved in #17
